src/pages/Home.jsx

```diff
@@ -0,0 +1,120 @@
+import { Routes, Route, NavLink } from 'react-router-dom';
+import { useState } from 'react';
+import WorldMap from 'react-svg-worldmap';
+import Grid from './Grid';
+import About from './About';
+import Navbar from '../components/Navbar';
+import styles from '../sass/pages/home.module.scss';
+
+const Home = () => {
+  const [header, setHeader] = useState('STATS BY COUNTRY');
+  const [menu, setMenu] = useState('bars');
+  const [open, setOpen] = useState(false);
```

Comment:
- [OPTIONAL] Kindly, you should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), what if you move your title or name of the country to the left of your header section as show bellow :arrow_down: 

||  Original [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))  ||
|-|-----------------------------------|-|
||![go1](https://user-images.githubusercontent.com/57562869/157501449-50bab51c-5494-447a-a950-0c1abbba504d.png)||

||![11111111111111](https://user-images.githubusercontent.com/57562869/158032898-c092d927-298b-4837-b04c-e58ef1059b0b.png)||